### PR TITLE
Add herbs2rest

### PR DESCRIPTION
**Pull request by gabrielmellooliveira**
## Add herbs2rest

1. New express routes generated by herbs2rest.
2. New function to delete a list. 
3. Front-end consume rest API.

### CODE REVIEW

**Comment by jhomarolo** [comment]
@gabrielmellooliveira this PR is ready to be analyzed?

**Comment by jhomarolo** [comment]
@gabrielmellooliveira this PR is broken, some lint error, could you please check this?

**Comment by gabrielmellooliveira** [comment]
@jhomarolo I committed the changes to improve the use of herbs2rest and correct lint error

File: backend/src/domain/usecases/_uclist.js [hunk 1]
```diff
@@ -4,7 +4,8 @@ module.exports = (injection) => {
         { usecase: require('./updateItem').updateItem(injection), tags: { group: 'Items' } },
         { usecase: require('./createList').createList(injection), tags: { group: 'List' } },
         { usecase: require('./getLists').getLists(injection), tags: { group: 'List' } },
+        { usecase: require('./getList').getList(injection), tags: { group: 'List' } },
```
Comment (dalssoft) [inline backend/src/domain/usecases/_uclist.js:7]: It should be removed, right?

File: backend/src/infra/api/rest/controllerList.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const controllerList = [
+  {
+    name: 'lists',
+    getAll: require('../../../domain/usecases/getLists').getLists,
```
Comment (dalssoft) [inline backend/src/infra/api/rest/controllerList.js:4]: Where is the example for GetByID?

File: frontend/src/pages/Home/Home.js [hunk 3]
```diff
@@ -10,8 +10,8 @@ function Home() {
         <div className={classes.toolbar} />
         <h1>Welcome!</h1>
         <p>
-          This is a sample Todo List graphQL API using Herbs using Buchu, Gotu,
-          Suma and Graphql with Apollo Server Express
+          This is a sample Todo List graphQL/rest API using Herbs using Buchu, Gotu,
```
Comment (dalssoft) [inline frontend/src/pages/Home/Home.js:13]: This text could be improved:
`This is a To Do List application with the purpose of demonstrate the use of Herbs and some important glues (GraphQL, REST, repository, etc)`

**Review by dalssoft — CHANGES_REQUESTED** [review]

File: frontend/src/pages/Home/Home.js (hunk 3 above)
Comment (gabrielmellooliveira) [reply frontend/src/pages/Home/Home.js]: Yes, I updated the text

File: backend/src/domain/usecases/_uclist.js (hunk 1 above)
Comment (gabrielmellooliveira) [reply backend/src/domain/usecases/_uclist.js]: Right, removed

File: backend/src/infra/api/rest/controllerList.js (hunk 2 above)
Comment (gabrielmellooliveira) [reply backend/src/infra/api/rest/controllerList.js:4]: I added, but as the focus of GetById is on the params field, so this is not the best example yet

**Review by dalssoft — APPROVED** [review]

**Pull request merged**